spec/index.md

```diff
@@ -2340,6 +2340,30 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `fetch_logs` {#ic-fetch_logs}
+
+Provides the logs of the canister including trap messages.
```

Comment:
> including trap messages

if these are automatically logged by the system, then devs should have a way to silence them so that they don't need to pay for the log memory consumed by those logs

Reply:
I suppose the first implementation does not have any means to control the logs verbosity to make it simple.
additionally the log size of 4KB is small enough not to care about the price. which can be (potentially) be even free.

Reply:
> the log size of 4KB is small enough

agreed, this is the order of magnitude taken by the canister history as well (which cannot be turned off either)